src/main/java/com/quadfour/QuadFourController.java

```diff
@@ -34,5 +36,12 @@ public String readOne(Model model) {
     	model.addAttribute("taskDTO", taskDTO);
         return "index";
     }
+    
+    @RequestMapping("/searchResults")
+    public String searchResults(@RequestParam(value="searchTerm", required=true)String searchTerm)
+    {
+    String enhancedSearchTerm = searchTerm + "";
```

Comment:
It looks like this enhancedSearchTerm variable is never used... unused code is clutter, which is technical debt.